src/applications/edu-benefits/10203/helpers.jsx

```diff
@@ -103,3 +103,8 @@ export function updateProgramDetailsSchema() {
     });
   };
 }
+
+export function phoneNumberFormatted(number) {
```

Comment:
I don't think this is needed as the phone number is stored in the state as only numbers.
See https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/widgets/PhoneNumberWidget.jsx#L18

Reply:
I added this helper to determine whether or not the alert should be shown given a valid number for 3c. `The alert is no longer displayed once the user enters a valid "Mobile phone number"`. I didnt see anywhere in the state that indicated valid vs invalid for that field.

Reply:
https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/definitions.js#L159

only validation on phone numbers should be a length check

`fullSchema10203.definitions.phone.minLength` might be the way to access the min length value to check against state data